src/styles/blocks/logo.css

```
@@ -40,12 +40,24 @@
 
 .logo__title {
     margin: 0;
-    font-weight: bold;
+    font-family: 'Dewi', sans-serif;
+    font-weight: 700;
```

```suggestion
    font-weight: bold;
```

Почему `700`, а не `bold`? У нас нет градаций 100–900 в подключении, только `normal` и `bold`.